src/main/java/edu/ucsb/cs56/ucsb_open_lab_scheduler/controllers/InstructorMenuController.java

```
@@ -89,4 +94,25 @@ public String getTutor(@PathVariable("id") long id, Model model, OAuth2Authentic
         return "instructorMenu/getTutors";
     }
 
+    @GetMapping("/InstructorMenu/downloadCSV/{id}")
+    public String exportCSV(@PathVariable long id, Model model, HttpServletResponse response,  OAuth2AuthenticationToken token, RedirectAttributes redirAttrs)
+            throws Exception {
```

This should throw an IO Exception